vita3k/motion/include/motion/state.h

```diff
@@ -19,14 +19,26 @@
 
 #include <motion/motion_input.h>
 
+#include <SDL3/SDL_sensor.h>
+
 #include <mutex>
 
 struct MotionState {
     std::mutex mutex;
     MotionInput motion_data;
     uint32_t last_counter = 0;
-    uint64_t last_gyro_timestamp = 0;
-    uint64_t last_accel_timestamp = 0;
+    uint64_t last_refresh_motion_timestamp = 0;
+
+    // gyroscope and accelerometer on the device itself
+    SDL_Sensor *device_gyro = nullptr;
+    SDL_Sensor *device_accel = nullptr;
```

Comment:
This can be changed to unique_ptr with custom destructor (SDL_CloseSensor) like this https://github.com/Vita3K/Vita3K/pull/3694/files#diff-8943f98aa2f4259c79bba5083cc50cf38d71d963e5ce72146807d339bc2ecd93R33-R47